src/Collection/DefaultIterator.php

```diff
@@ -316,7 +316,7 @@ public function next()
     {
         next($this->entries);
         $nextEntry = current($this->entries);
-        $this->current = $nextEntry['resource'];
+        $this->current = $nextEntry ? $nextEntry['resource'] : null;
```

Comment:
I think it would be better to use:
```php
isset($nextEntry['resource']) ? $nextEntry['resource'] : null;
```

Reply:
Agreed! I'll change it up and look at the unit tests. Thanks.

Reply:
I think it's covered - https://coveralls.io/builds/29323361/source?filename=src/Collection/DefaultIterator.php#L319 

If we add a PHP 7.4 configuration block to Travis CI to the mainline branch we should hopefully be able to see the error being emitted, can then rebase this PR onto that and see it change the build to passing?

Reply:
Yes, you are right. Thanks also for the other PR 👍